src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let score = 1;
+
+  const getSum = (x) => (y) => {
```

Comment:
Let's use more readable syntax. And also, no need to create variable if you return it right away.
```suggestion
  return (x) => {
    return (y) => {
```